src/thermo/src/element/4C_thermo_element.hpp

```diff
@@ -27,571 +27,573 @@ namespace Core::FE
   class Discretization;
 }  // namespace Core::FE
 
-namespace Thermo
+namespace Discret
```

Comment:
Isn't `Thermo` the namespace we want in the future? `Discret::Elements` is conceptually not a modular namespace